bot/plotting/comparison_generator.py

```diff
@@ -123,10 +118,13 @@ def comparison_generator(
         choice_list = ["experiment", "no experiment"]
         choice = random.choice(choice_list)
 
-    if choice == "no experiment":
+    while True:
+        try:
+
+            if choice == "no experiment":
+
+                is_experiment = False
 
-        while True:
-            try:
                 s = pybamm.BatchStudy(
```

Comment:
Very minor, but it would be better to give this a more informative name than `s`. Maybe `batch_study`? You can change this in a future PR though.